syntax = "proto3";

package protocol.db;

enum E_PacketID
{
	LOGIN_REQUSET = 0;
	LOGIN_RESPONSE = 1;
}
enum E_RespCode
{
	OK = 0;

	LOGIN_INVALID_ID = 10;
	LOGIN_INVALID_PW = 11;
}

message Login_Req
{
	int32 serial = 1;
	string id = 2;
	string passwd = 3;
}

message Login_Resp
{
	int32 serial = 1;
	E_RespCode resp = 2;
	string nickname = 3;
}